docs/HowTo/Interact/APIs/Authentication.md

```diff
@@ -34,17 +35,21 @@ The `toml` file defines user details and the JSON-RPC methods to which they have
     [Users.username1]
     password = "$2a$10$l3GA7K8g6rJ/Yv.YFSygCuI9byngpEzxgWS9qEg5emYDZomQW7fGC"
     permissions=["net:*","eth:blockNumber"]
+    privacyPublicKey="U7ANiOOd5L9Z/dMxRFjdbhA1Qragw6fLuYgmgCvLoX4="
     
     [Users.username2]
     password = "$2b$10$6sHt1J0MVUGIoNKvJiK33uaZzUwNmMmJlaVLkIwinkPiS1UBnAnF2"
     permissions=["net:version","admin:*"]
+    privacyPublicKey="quhb1pQPGN1w8ZSZSyiIfncEAlVY/M/rauSyQ5wVMRE="
     ```
 
 Each user requiring JSON-RPC access is listed with: 
 
 * Username. `Users.` is mandatory and followed by the username. That is, replace `<username>` in `[Users.<username>]` with the username being defined. 
 * Hash of the user password. Use the [`password hash`](../../../Reference/CLI/CLI-Subcommands.md#password) subcommand to generate the hash. 
-* [JSON-RPC permissions](#json-rpc-permissions). 
+* [JSON-RPC permissions](#json-rpc-permissions).
+* Optional. The tenant's Orion public key using `privacyPublicKey`. Only used
+for multi-tenancy. 
```

Comment:
Might be worth adding a link to the multi-tenancy concept page

Reply:
Done.